src/frontends/tensorflow/src/translate_session.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/frontend/input_model.hpp"
+#include "openvino/frontend/tensorflow/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+using CachedBodyModelsType = std::unordered_map<std::string, std::shared_ptr<const ov::Model>>;
+using TelemetryDataType = std::vector<std::pair<std::string, std::string>>;
+
+/// For one call of convert and decode method of Frontend, it creates one TranslateSession object to save data for the
+/// translation session: telemetry statistics, cache of convrted body graph models, operation translators (including
+/// extensions) registered for this translation session.
+class TranslateSession {
+public:
+    TranslateSession(const ov::frontend::InputModel::Ptr& input_model,
+                     const std::shared_ptr<TranslatorDictionaryType>& translator_map,
+                     const std::string& model_name,
+                     bool failed_fast,
```

Comment:
The name of this parameter was `fail_fast` which meant that in case of error - fail. New name means something slightly different. And why flag `no_conversion` disappeared?

Reply:
renamed back to `fail_fast`. `no_conversion` flag is no longer needed because it serves to distinguish `convert`  and `decode` methods for which different translators dictionary used.